ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/multisplitpanel/panel/SubPanelViewImpl.java

```diff
@@ -268,16 +268,33 @@ public void remove() {
   private void removeWidgetFromUI(WidgetToShow widget) {
     final Tab tab = widgets2Tabs.remove(widget);
     if (tab != null) {
+      final int removedTabIndex = tabsPanel.getWidgetIndex(tab);
+
       tabsPanel.remove(tab);
       widgetsPanel.remove(widget.getWidget());
-
       tabs2Widgets.remove(tab);
 
       // remove item from drop-down menu
       final MenuItemWidget listItemWidget = widgets2ListItems.remove(widget);
       if (listItemWidget != null) {
         menu.removeListItem(listItemWidget);
       }
+
+      if (tab == selectedTab) {
+        if (tabsPanel.getWidgetCount() > 1) {
```

Comment:
Here we can replace, I think 
```
 if (tab == selectedTab) {
        if (tabsPanel.getWidgetCount() > 1) {
```
> 
`if (tab == selectedTab && tabsPanel.getWidgetCount() > 1) {`

Reply:
Done. Thanks